### ISSUE TITLE
[A11y] The header's orange "search" button does not have a contrast ratio of 3:1 or higher

### ISSUE THREAD

**Issue by loic-sharma** [NuGet/NuGetGallery#8265]
Follow up to https://github.com/NuGet/NuGetGallery/issues/8195

AzDO work item: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1188278
